OneCommunityGlobal/HGNRest
# Sushmitha project controller unit and integration test

# Description
PR adds integration tests and updates the unit tests for projectController.js

## Main changes explained:
- Added integration test file named: projectController.test.js
- Update unit test file named projectController.spec.js
…

## How to test:
1. check into the current branch
2. do npm install
3. To run the unit tests: do npm run test projectController
4. To run the integration tests: do npm run test projectRouter
5. All test cases should pass
